static/styles/rendered_markdown.css

```diff
@@ -673,6 +677,8 @@ pre {
 pre code {
     overflow-x: scroll;
     white-space: pre;
```

Comment:
I think Bootstrap had `white-space pre-wrap` specified after `pre`; does this represent a change in how long lines are wrapped inside these blocks?

Reply:
Currently, all instances of `pre code` also have the `rendered_markdown` class in the web-app. And that has a more specific rule of `white-space: inherit;`, which inherits the rule from the parent class (`pre`) and that's set by Bootstrap as `white-space: pre;` currently.

Does it make sense to also do the Bootstrap `pre` element changes with these change as well?

Noting that even if we did have a case of this in the web-app, this current rule here would already override the Bootstrap one of `pre-wrap`. I checked this manually by making one of the linkifier code examples in the Org settings overly to be inside a `pre` element.

Reply:
I'll also note that a code block without the `rendered_markdown` class would be very broken in dark theme currently.